src/MOI_wrapper.jl

```diff
@@ -341,19 +412,40 @@ end
 function MOI.get(
     optimizer::Optimizer,
     attr::MOI.ConstraintPrimal,
-    ci::MOI.ConstraintIndex,
+    ci::MOI.ConstraintIndex{MOI.VectorAffineFunction{Float64}},
 )
     MOI.check_result_index_bounds(optimizer, attr)
-    return optimizer.sol.slack[MOI.Utilities.rows(optimizer.cones, ci)]
+    return optimizer.sol.slack[MOI.Utilities.rows(optimizer.cones_real, ci)]
+end
+function MOI.get(
+    optimizer::Optimizer,
+    attr::MOI.ConstraintPrimal,
+    ci::MOI.ConstraintIndex{MOI.VectorAffineFunction{ComplexF64}},
+)
+    MOI.check_result_index_bounds(optimizer, attr)
+    return optimizer.sol.slack[optimizer.complex_offset .+ MOI.Utilities.rows(
+        optimizer.cones_complex,
+        ci,
+    )]
+end
+function MOI.get(
+    optimizer::Optimizer,
+    attr::MOI.ConstraintDual,
+    ci::MOI.ConstraintIndex{MOI.VectorAffineFunction{Float64}},
+)
+    MOI.check_result_index_bounds(optimizer, attr)
+    return optimizer.sol.x[MOI.Utilities.rows(optimizer.cones_real, ci)]
```

Comment:
You probably need something like
```suggestion
    return convert(
        Vector{Float64},
        optimizer.sol.x[MOI.Utilities.rows(optimizer.cones_real, ci)],
    )
```